crates/bevy_ecs/src/component.rs

```diff
@@ -286,7 +287,7 @@ impl ComponentInfo {
 /// Given a type `T` which implements [`Component`], the `ComponentId` for `T` can be retrieved
 /// from a `World` using [`World::component_id()`] or via [`Components::component_id()`]. Access
 /// to the `ComponentId` for a [`Resource`] is available via [`Components::resource_id()`].
-#[derive(Debug, Copy, Clone, Hash, Ord, PartialOrd, Eq, PartialEq)]
+#[derive(Debug, Copy, Clone, Hash, Ord, PartialOrd, Eq, PartialEq, Reflect)]
```

Comment:
Could you also add the appropriate registrations (e.g. `#[reflect(Hash, PartialEq)]`)?

Same for the other items.

Reply:
Sure, should i add `Clone` as well?

Reply:
I don't think there is one for `Clone` 🤔